package_scanner.nim

```diff
@@ -116,6 +117,10 @@ proc check(): int =
       echo "E: ", name, " has no license"
       result.inc()
 
+    elif pdata["url"].str.normalize.startsWith("git://github.com/"):
```

Comment:
I have no idea which hosters support https and if they use the same URL. I know that it would be problematic for self-hosted repositories, haven't used any of the others.